packages/squiggle-lang/src/fr/table.ts

```diff
@@ -45,20 +46,20 @@ export const library = [
             data: value,
             columns: columns.map(({ fn, name }) => ({
               fn,
-              name: name ?? undefined,
+              name: name ?? args.name() ?? undefined,
```

Comment:
Won't this cause identical names for all columns if `name` in `columns` config is not set? Is so, is it useful?
I can see the benefit of using `fn`'s name if `fn` is boxed, or using 1/2/3/4 numbers.

While I tried to test it, I encountered a bug, but I'm not sure what's causing it: https://squiggle-website-git-boxed-values-plus-quantified-uncertainty.vercel.app/playground?v=dev#code=eNp1kEFrwzAMhf%2FKw6eWeYWt2yWwQbaed8otzsBNlNbMccBRDiPNf589d20GGxjrCel7EppEQ60eLb%2F2DYlMNJo1nlDeSTxIPFa4fUahDxunO1opscuLXIm1cspxaCv03tKm0x%2Bh5oApfgAbtpRBiRc9mDo1KSFTMQ44y7q3Y%2BeGDGXKgwFal2E60Qk0S8Sh0edt7PbklcAs%2F%2Bh0ve%2B0XRHecS8Rw3a9gHdm4P%2FQkgKQXrVAcu%2F152%2FmR%2BAKBx%2FjB84iPlDduyZI3CCcjo%2FGn7OwUtuPno8p3WK%2BumKxZM2XE%2BEyuPqOs3Lh4mL%2BAndmeyQ%3D

Is `frForceBoxed` broken?

Reply:
This is a mistake, my bad

Reply:
I'm still confused why my playground link doesn't work.